### ISSUE TITLE
collectd logs on duplicate values

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla#3190]
*Installation details*
Scylla version (or git commit hash): 2.1.rc3
scylla-2.1.rc3-0.20180206.cbb3b959e.el7.centos.x86_64

```
eb 06 17:11:08 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-2/gauge-queued_reads-reads; value time = 1517937068.000; last cache update = 1517937068.000;
Feb 06 17:11:08 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-2/gauge-queued_reads-reads; value time = 1517937068.000; last cache update = 1517937068.000;
Feb 06 17:11:08 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-0/gauge-queued_reads-reads; value time = 1517937068.000; last cache update = 1517937068.000;
Feb 06 17:11:08 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-0/gauge-queued_reads-reads; value time = 1517937068.000; last cache update = 1517937068.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-3/gauge-active_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-3/gauge-active_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-3/gauge-active_reads_memory_consumption-reads; value time = 1517937071.000; last cache update = 1517937071.
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-3/gauge-active_reads_memory_consumption-reads; value time = 1517937071.000; last cache update = 1517937071.
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-3/gauge-queued_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-3/gauge-queued_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-1/gauge-active_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-1/gauge-active_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-1/gauge-active_reads_memory_consumption-reads; value time = 1517937071.000; last cache update = 1517937071.
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-1/gauge-active_reads_memory_consumption-reads; value time = 1517937071.000; last cache update = 1517937071.
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-1/gauge-queued_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-1/gauge-queued_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-2/gauge-active_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-2/gauge-active_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-2/gauge-active_reads_memory_consumption-reads; value time = 1517937071.000; last cache update = 1517937071.
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-2/gauge-active_reads_memory_consumption-reads; value time = 1517937071.000; last cache update = 1517937071.
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-2/gauge-queued_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-2/gauge-queued_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-0/gauge-active_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-0/gauge-active_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-0/gauge-active_reads_memory_consumption-reads; value time = 1517937071.000; last cache update = 1517937071.
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-0/gauge-active_reads_memory_consumption-reads; value time = 1517937071.000; last cache update = 1517937071.
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-0/gauge-queued_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
Feb 06 17:11:11 ip-172-30-0-84 collectd[3208]: uc_update: Value too old: name = ip-172-30-0-84/database-0/gauge-queued_reads-reads; value time = 1517937071.000; last cache update = 1517937071.000;
```

**Comment by amnonh** [comment]
Need to backport:
https://github.com/scylladb/scylla/commit/a0a1961b6dbfeb784294cbaf02e087f73974ad75

**Comment by slivne** [comment]
fixed by d306c40507afea0ddfd570697c0a491816f1030f
